### PR TITLE
Taskreview

### PR DESCRIPTION
**Pull request by Bart-Ilg**
Merging in task review page

### CODE REVIEW

File: src/Components/TaskReview/QuestionReview.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        return '#27AE60';
+      }
+      return '#EB5757';
+   }
+
+   function displayCorrectAnswer(){
+      if(type == 'MC_QUESTION' && questionAndAnswer.answer.pointsAwarded == 0){
+         return (
+            <h2 className="questionReviewText">Correct Answer: { questionAndAnswer.answer.correctAnswer}</h2>
+         );
```
Comment (samiur98) [inline src/Components/TaskReview/QuestionReview.js:22]: Looks good.

File: src/Components/TaskReview/TaskReview.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.continueButtonContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.questionReviewContainer {
+  border: 4px solid white;
+}
+
```
Comment (samiur98) [inline src/Components/TaskReview/TaskReview.css:10]: Good job CSS.

File: src/Components/TaskReview/TaskReviewStats.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+   function getScore(){
+     var n = 0;
+     for(var i = 0; i < submission.questionAndAnswers.length; i++){
+      if(submission.questionAndAnswers[i].answer.pointsAwarded !== 0){
+        n++;
+      }
+     }
+     return n
+   }
+
```
Comment (samiur98) [inline src/Components/TaskReview/TaskReviewStats.js:17]: Good Code.

File: src/gqlQueries.js [hunk 4]
```diff
@@ -304,10 +302,19 @@ export const SUBMIT_TASK = gql`
...
-          id
-          description
-          points
-          feedback
+          ... on McQuestion {
+            id
+            description
+            points
+            feedback
+            answers
```
Comment (samiur98) [inline src/gqlQueries.js:310]: Good job on the query.

**Review by samiur98 — APPROVED** [review]
Looks good to be merged.

File: src/gqlQueries.js [hunk 5]
```diff
@@ -75,7 +75,6 @@ export const GET_TASK = gql`
       requirements {
         id 
         description
-        isComplete
```
Comment (tddumont) [inline src/gqlQueries.js:78]: Good to remove this, since it isn't used.

File: src/Components/MultiSelect.js [hunk 6]
```diff
@@ -13,13 +8,12 @@ function MultiSelect() {
       setAnswer( e.target.value);
    }
 
-   const onSubmit = async (event) => {
+   const onSubmit = async () => {
     alert("Submitted!");
```
Comment (tddumont) [inline src/Components/MultiSelect.js:12]: I'm curious if this needs to be a function, since it is just one line.

**Review by tddumont — APPROVED** [review]
Great work! The new code looks very organized!

**Pull request merged**